azure_functions_devops_build/yaml/yaml_manager.py

```diff
@@ -113,13 +132,3 @@ def _dotnet_dependencies(self):
         dependencies.append("    modifyOutputPath: true")
         dependencies.append("    zipAfterPublish: false")
         return dependencies
-
-    def _java_dependencies(self):
```

Comment:
Best to have this in a separate PR (rationale above).

Reply:
Thanks, same as above. Reverted.